pocs/tests/test_database.py

```diff
@@ -4,24 +4,24 @@
 from pocs.utils.logger import get_root_logger
 
 
-def test_insert_and_get_current(db):
+def test_insert_and_no_permanent(db):
```

Comment:
The PR description explains but basically if left in the current order the DB would have to be cleared before second test otherwise a record exists because it was stored permanently in first test. Rearranging allows them both to proceed without manually forcing. 

More natural to test w/o permanent first and then do a permanent.